ui/components/Filters.js

```diff
@@ -435,7 +447,7 @@ function MesheryFilters({ updateProgress, enqueueSnackbar, closeSnackbar, user,
         <YAMLEditor filter={selectedRowData} onClose={resetSelectedRowData()} onSubmit={handleSubmit} />
       )}
       <MUIDataTable
-        title={<div className={classes.tableHeader}>Filters</div>}
+        title={<div className={classes.tableHeader}>Filters?</div>}
```

Comment:
I might be missing context, I'm trying to think of why we would want a question mark here, though...￼

Reply:
Ahh... sorry about that, I added that at the very beginning while playing around with the code. I'll remove that :D

Reply:
No sweat. Resolve this comment when you have removed it.